### PR TITLE
[TF FE] Support ApproximateEqual operation for TensorFlow

### PR DESCRIPTION
**Pull request by ViswanathBalla22**
Hello OpenVINO community!
This is my first contribution to the OpenVINO project, and I'm excited to be a part of this amazing community. I've implemented support for the `ApproximateEqual` operation for TensorFlow, addressing [link to the GitHub issue].
## Changes Made -
I added a new test case (`test_tf_approximate_equal.py`) in the `layer_tests/tensorflow_tests` directory to ensure proper support for the `ApproximateEqual` operation. The test validates the functionality on different devices and precisions....

I'm looking forward to feedback from the community and reviewers. If there are any improvements or adjustments needed, I'm eager to collaborate and make the necessary changes.
Looking forward to contributing more to OpenVINO in the future!

### CODE REVIEW

File: src/frontends/tensorflow/src/op/approximateequal.cpp [hunk 1]
```diff
@@ -0,0 +1,40 @@
+// approximateequal.cpp
+#include <ngraph/node.hpp>
+#include <ngraph/opsets/opset1.hpp>
+#include <ngraph/op/util/op_types.hpp>
+#include <ngraph/validation_util.hpp>
```
Comment (rkazants) [inline src/frontends/tensorflow/src/op/approximateequal.cpp:5]: please avoid use of `ngraph`, this release we tootally remove it.
Just see this PR as an example https://github.com/openvinotoolkit/openvino/pull/19007 where we use only ov headers

**Review by rkazants — COMMENTED** [review]
please update `https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/tensorflow/docs/supported_ops.md` with supported `ApproximateEqual` operation. Without it, CI will fail.

File: src/frontends/tensorflow/src/op/approximateequal.cpp [hunk 2]
```diff
@@ -0,0 +1,40 @@
+// approximateequal.cpp
```
Comment (rkazants) [inline src/frontends/tensorflow/src/op/approximateequal.cpp:1]: please rename source file to `approximate_equal.cpp` to have it more accurate.

File: src/frontends/tensorflow_common/include/common_op_table.hpp [hunk 3]
```diff
@@ -146,6 +146,7 @@ OP_CONVERTER(translate_softmax_op);
 OP_CONVERTER(translate_space_to_depth_op);
 OP_CONVERTER(translate_split_op);
 OP_CONVERTER(translate_split_v_op);
+OP_CONVERTER(ApproximateEqual);
```
Comment (rkazants) [inline src/frontends/tensorflow_common/include/common_op_table.hpp:149]:
```suggestion
OP_CONVERTER(translate_approximate_equal_op);
```

File: src/frontends/tensorflow/src/op/approximateequal.cpp [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+#include <ngraph/node.hpp>
+#include <ngraph/opsets/opset1.hpp>
+#include <ngraph/op/util/op_types.hpp>
+#include <ngraph/validation_util.hpp>
+
+using namespace std;
+using namespace ov;
+using namespace ov::opset1;
```
Comment (rkazants) [inline src/frontends/tensorflow/src/op/approximateequal.cpp:9]: please avoid use of opset header and include concrete headers for concrete operations you are using for your decomposition below.

**Comment by ViswanathBalla22** [comment]
ok @rkazants thanks for suggestion I will work on it

**Comment by ViswanathBalla22** [comment]
Hi @rkazants 

I apologize for the oversight. I mistakenly submitted the pull request without making the necessary modifications. Could you please avoid reviewing or merging it? I will make the required changes and resubmit the corrected version shortly.
Thank you for your understanding.

**Pull request closed without merging**